contents/load_save.md

```diff
@@ -0,0 +1,209 @@
+## Load and Save Files {#sec:load_save}
+
+Having only data inside Julia programs and not being able to load or save it would be very limiting.
+Therefore, we start by mentioning how to store files to disk and how to load files from disk.
+We focus on CSV, see @sec:csv, and Excel, see @sec:excel, since those are the most common.
+
+### CSV {#sec:csv}
+
+Comma-separated value (CSV) files are are ridiculously effective way to store tables.
+Unlike many competing data storage solutions, CSV files have two advantages.
+Firstly, it does exactly what the name indicates it does, namely storing values by separating the columns by commas.
+This abbreviation is, also, used as the file extension.
+So, be sure that you save your files as "myfile.csv".
+To demonstrate how a CSV file looks, we can install the CSV package:
+
+```
+julia> ]
+
+pkg> add CSV
+```
+
+and load it via
+
+```
+using CSV
+```
+
+We can, now, take some data:
+
+```jl
+sco("
+grades_2020()
+"; process=without_caption_label)
+```
+
+and read it from a file after writing it
```

Comment:
This is somewhat counter-intuitive. I am reading data as a CSV file that I already had in Julia and wrote as a CSV file?
Wouldn't be simpler and less confusing to just start with `CSV.read()` and explain the difference between `CSV.File()` and `CSV.read()`?

Reply:
Currently, it is more in line with the next section on Excel. From a programmers perspective, I wouldn't think that either writing or loading comes first. Some people write first, when they start programming, and some people load data first.